src/main/java/net/sf/marineapi/ais/message/AISPositionInfo.java

```diff
@@ -36,9 +36,21 @@
 	 * Returns the longitude in degrees.
 	 */
 	double getLongitudeInDegrees();
-	
+
 	/**
 	 * Returns the latitude in degrees.
 	 */
 	double getLatitudeInDegrees();
+
+	/**
+	 * Returns true if longitude is available in the message.
+	 * If false, getLongitudeInDegrees may return an out-of-range value.
+	 */
+	boolean isLongitudeAvailable();
```

Comment:
I'd suggest to use "has" instead of "is" for shorter notation and readability, e.g. in this case `hasLongitude()`.

Reply:
Check.